spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -32,11 +34,16 @@
  * register Gravitino catalogs to Spark.
  */
 public class GravitinoDriverPlugin implements DriverPlugin {
+
   private static final Logger LOG = LoggerFactory.getLogger(GravitinoDriverPlugin.class);
 
   private GravitinoCatalogManager catalogManager;
-  private static final String[] GRAVITINO_DRIVER_EXTENSIONS =
-      new String[] {IcebergSparkSessionExtensions.class.getName()};
+  private List<String> gravitinoDriverExtensions = new ArrayList<>();
+  private boolean supportsIceberg = false;
+
+  @VisibleForTesting
+  static final String ICEBERG_SPARK_EXTENSIONS =
+      "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions";
```

Comment:
seems can directly use `IcebergSparkSessionExtensions.class.getName()`?  it can build successfully.

Reply:
the package maybe not exists at runtime, I will get a try.

Reply:
spark connector failed to startup without Iceberg runtime jar
```
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/iceberg/spark/extensions/IcebergSparkSessionExtensions
	at com.datastrato.gravitino.spark.connector.plugin.GravitinoDriverPlugin.<clinit>(GravitinoDriverPlugin.java:46)
	at com.datastrato.gravitino.spark.connector.plugin.GravitinoSparkPlugin.driverPlugin(GravitinoSparkPlugin.java:17)
```

Reply:
> spark connector failed to startup without Iceberg runtime jar if using IcebergSparkSessionExtensions.class.getName()
> 
> ```
> Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/iceberg/spark/extensions/IcebergSparkSessionExtensions
> 	at com.datastrato.gravitino.spark.connector.plugin.GravitinoDriverPlugin.<clinit>(GravitinoDriverPlugin.java:46)
> 	at com.datastrato.gravitino.spark.connector.plugin.GravitinoSparkPlugin.driverPlugin(GravitinoSparkPlugin.java:17)
> ```

got it.